rust-bindings/bls-dash-sys/build.rs

```diff
@@ -127,7 +164,8 @@ fn main() {
     cc.files(cpp_files)
         .includes(&include_paths)
         .cpp(true)
-        .flag_if_supported("-std=c++14");
+        .flag_if_supported("-std=c++14")
+        .target(&env::var("TARGET").unwrap());
```

Comment:
the `cc` crate does support using the correct compiler based on the `CC_*` and `AR_*` environment variables, but it needs to know the target.